tests/functional/views/lti/deep_linking_test.py

```diff
@@ -0,0 +1,122 @@
+import time
+
+import oauthlib.common
+import oauthlib.oauth1
+import pytest
+
+from lms.resources._js_config import JSConfig
+
+
+class TestDeepLinkingLaunch:
+    def test_basic_lti_launch_canvas_deep_linking_url(
+        self,
+        do_deep_link_launch,
+        lti_params,
+        application_instance,
+        get_client_config,
+        environ,
+    ):
+        response = do_deep_link_launch(post_params=lti_params, status=200)
```

Comment:
Very simple test, similar to the basic launch ones.

Make a request, assert the contents of the response's config.